contents/product-engineers/what-is-multivariate-testing-examples.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Multivariate testing, explained (with examples)
+date: 2023-11-15
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+tags:
+  - Growth engineering
+  - Product engineers
+  - AB testing
+---
+
+## What is multivariate testing?
+
+Multivariate testing is when you test several different combinations of changes to determine which performs the best.
+
+It's distinct from A/B testing, where you only test two variants – a control and a single variant you want to test against it.
+
+![A/B testing vs multivariate testing](../images/blog/multivariated-testing-explained/ab-vs-mvt.png)
+
+Here's an example:
+
+Imagine you want to test how the color _and_ placement of a button affects your conversion rate. In A/B testing you'd have to run three separate tests, one after the other, to determine which change had an impact.
+
+In multivariate testing you simply create one test with four different variants:
+
+1. The control variant.
+2. The variant with only the color changed.
+3. The variant with only the placement changed.
+4. The variant with both the color and the placement changed.
+
+The results from the multivariate test show not only which variant performs the best, but also how each factor affected the outcome.
+
+## Benefits of multivariate testing
+
+### 1. It saves you time
+
+Running multiple A/B tests at the same risks them interfering with each other. As a result, you usually need to wait for each test to finish before starting the next one.
+
+Multivariate tests enable you to test multiple combinations at the same time, meaning you don't have to wait for many consecutive tests to finish.
+
+### 2. It reveals how different elements interact with each other
+
+Multivariate testing is not just about looking at which variant is the best; it's about understanding the weight each factor had in that success.
+
+For example, let's say you're testing changes to the user dashboard of a B2B SaaS app. You decide to test how both the position of the navigation menu and the text of the buttons affect user engagement.
+
+In isolation, changing the button text might affect how well users understand your app. Similarly, moving the navigation menu from the top to the side could influence their ability to navigate it.
+
+However, when you combine and test these changes together, you may discover that their combined effect is not necessarily greater than each individual change.
+
+Multivariate testing enables you to uncover insights like these.
```

Comment:
Instead of this point (which is a bit repetitive), maybe you can say "this helps you make the next set of optimizations to these features". If you know what factors are most important, it helps you make future decisions.